tcms/issuetracker/types.py

```diff
@@ -378,3 +378,14 @@ def report_issue_from_testcase(self, caserun):
             url += '/'
 
         return url + '/issues/new?' + urlencode(args, True)
+
+
+class Dummy(IssueTrackerType):
```

Comment:
Rename to LinkOnly or something like that and we're good to merge. Nobody in their right mind will select a "Dummy" option for a production instance.